### PR TITLE
API-3435 - Add Claims Attributes API in Dev

### PR DESCRIPTION
**Pull request by rtravitz**
### Description

This PR supports [API-3435](https://vajira.max.gov/browse/API-3435). It adds a scaffold for Claims Attributes to the Developer Portal but enables it in `dev` only. The Claims Attributes API is still getting deployed, so they do not yet have a publicly accessible documentation link. More work will be done on this section and a real Open API Spec will be completed before this is enabled in higher environments. 

The description field still needs to be updated with real copy, but I'm starting this PR now to get review bundles in S3.

### Process

This is a data change and not a structural change. It does not affect any of the listed categories.

### CODE REVIEW

File: src/apiDefs/data/benefits.ts [hunk 1]
```diff
@@ -52,6 +53,21 @@ const benefitsApis: APIDescription[] = [
     urlFragment: 'loan_guaranty',
     vaInternalOnly: false,
   },
+  {
+    description: 'Improve claim routing',
+    docSources: [
+      {
+        openApiUrl:
+          'https://gist.githubusercontent.com/rtravitz/599f6d35e4a763d85e2a1f49f8d031a3/raw/e0470da0ef1b6cb1e5d3a99f09c9d60638f29211/claims-attributes.json',
```
Comment (mjlumetta) [inline src/apiDefs/data/benefits.ts:61]: lol kudos for coming up with this workaround 😂

**Review by mjlumetta — DISMISSED** [review]
🚀 ✨

**Review by alexandervalencia — APPROVED** [review]
LGTM, thanks again for walking through this with me 🚀

**Review by TS000 — DISMISSED** [review]
LGTM 🏋️‍♂️

**Review by alexandervalencia — APPROVED** [review]

**Pull request merged**